### ISSUE TITLE
Regression in "group": cycles on its own, does not respect button_{next,prev}

### ISSUE THREAD

**Issue by maximbaz** [ultrabug/py3status#1756]
Since https://github.com/ultrabug/py3status/commit/910b74d39bbd3ca43a93426c72ce13a610216691 I have a very weird behavior:

- Modules in group cycle on their own, i.e. I see the first module, then suddenly the second module shows up, then the first one again, etc.
- Configured button_{next,prev} have no effect, instead they scroll only on mouse wheel up and down

![group](https://user-images.githubusercontent.com/1177900/55198504-85ab3b00-51b6-11e9-8f78-9642c8a693f0.gif)

I confirm that https://github.com/ultrabug/py3status/commit/910b74d39bbd3ca43a93426c72ce13a610216691 is the first bad commit

Here's my exact config:

```
group cpu {
  button_next = 1
  button_prev = 0

  sysdata {
    cache_timeout = 1
    thresholds = { 'cpu': [(0, 'good'), (25, 'degraded'), (80, 'bad')] }
    format = '[\?color=cpu 󰀐   {cpu_usage:.0f}%]'
  }

  conky {
    format = '󰀐  {top name 1} {top cpu 1}%'
  }
}

group ram {
  button_next = 1
  button_prev = 0

  sysdata {
    cache_timeout = 1
    thresholds = { 'mem': [(0, 'good'), (60, 'degraded'), (80, 'bad')] }
    format = '[\?color=mem 󰀌   {mem_used_percent:.0f}%]'
  }

  conky {
    format = '󰀌  {top_mem name 1} {top_mem mem 1}%'
  }
}
```

**Comment by lasers** [comment]
Try https://github.com/ultrabug/py3status/tree/group-cycling 

Too focused on non-urgent that I didn't check for cycle value.

**Comment by maximbaz** [comment]
Yes sir, it fixes the issue! Please feel free to merge it in master, you have my approval 👍

**Comment by lasers** [comment]
Closed via #1757.

**Comment by kozross** [comment]
Will there be a bugfix release?

**Comment by maximbaz** [comment]
cc @ultrabug

**Comment by lasers** [comment]
Temporary fix. You can set `cycle` with a large number like `21600` (6 hours).

**Comment by kozross** [comment]
@lasers Sure, will do that for the moment.

**Comment by ultrabug** [comment]
Thanks a lot guys, I've released 3.18 !
